### PR TITLE
Add Finnish translations

### PR DESCRIPTION
**Pull request by jonnelafin**
Remove the compiled translation

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -7,6 +7,7 @@ de
 es
 et
 fr
+fi
```
Comment (daniellandau) [inline po/LINGUAS:10]: Pitäisi olla fi_FI

Comment (jonnelafin) [reply po/LINGUAS:10]: Or should it? I submitted a translation for guake recently and they noticed it shoud be the opposite according to the ISO standard (at least according to [wikipedia](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)) I found two other projects where contributors were complaining on the use of fi_FI: [nextcloud](https://github.com/nextcloud/appstore/issues/479) and [woocommerence](https://github.com/woocommerce/woocommerce/issues/6302).

My commit however contradicts with itself, using fi once and fi_FI on the actual po file. I can change one of these to match the other. I myself have no preference. Do you want for this project to use fi or fi_FI?

P.S: Olisin voinu kirjottaa kans suomeks mut ajattelin et parempi kirjottaa englanniks jos joku muu törmää tähän tulevaisuudessa.

Comment (daniellandau) [reply po/LINGUAS:10]: Either way is fine for me.

Comment (jonnelafin) [reply po/LINGUAS:10]: I amended a commit changing both to just "fi"

**Pull request merged**